modules/installation-azure-config-yaml.adoc

```diff
@@ -37,22 +41,37 @@ This sample YAML file is provided for reference only. You must obtain your `inst
 apiVersion: v1
 baseDomain: example.com <1>
 controlPlane: <2>
+ifndef::azure-restricted[]
   hyperthreading: Enabled <3> <4>
+endif::azure-restricted[]
+ifdef::azure-restricted[]
+  hyperthreading: Enabled <3>
+endif::azure-restricted[]
   name: master
   platform:
     azure:
       encryptionAtHost: true
       ultraSSDCapability: Enabled
       osDisk:
+ifndef::azure-restricted[]
         diskSizeGB: 1024 <5>
+endif::azure-restricted[]
+ifdef::azure-restricted[]
+        diskSizeGB: 1024 <4>
+endif::azure-restricted[]
         diskType: Premium_LRS
         diskEncryptionSet:
           resourceGroup: disk_encryption_set_resource_group
           name: disk_encryption_set_name
           subscriptionId: secondary_subscription_id
       type: Standard_D8s_v3
   replicas: 3
+ifndef::azure-restricted[]
 compute: <2>
+endif::azure-restricted[]
+ifdef::azure-restricted[]
+compute: <5>
+endif::azure-restricted[]
```

Comment:
The description of <2> can be used here. No need to add ifdef and ifndef for `azure-restricted`.